### PR TITLE
Introduce dashed progress bar

### PR DESCRIPTION
**Pull request by kdzwinel**
dashed PIBIs!

### CODE REVIEW

File: components/dashed-progress-bar/src/js/main.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    this.ctx.beginPath();
+    this.ctx.arc(this.radius + this.margin, this.radius + this.margin, this.radius, start, end, 0);
+    this.ctx.stroke();
+  }
+
+  animate(callback) {
+    this.ctx.clearRect(0, 0, this.canvas.width, this.canvas.height);
+
+    let t = 0;
```
Comment (vergilius) [inline components/dashed-progress-bar/src/js/main.js]: this could be more readable :)

**Comment by vergilius** [comment]
image

👍

File: components/dashed-progress-bar/src/js/main.js (hunk 1 above)
Comment (matzimowski) [inline components/dashed-progress-bar/src/js/main.js]: T means a lot to me!

File: components/dashed-progress-bar/src/js/main.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    //draw gray tiles
+    for (let i = 0; i < this.tiles; i++) {
+      this._drawTile(i, '#dde1e4', this.endWidth - 1);
```
Comment (aju) [inline components/dashed-progress-bar/src/js/main.js]: I would make '#dde1e4' configurable

**Comment by aju** [comment]
Cool!

**Comment by kdzwinel** [comment]
@vergilius updated variable name from 't' to 'time'

@aju I made the grey tile color configurable

Mergin' !

**Pull request merged**